fe/fe-core/src/main/java/com/starrocks/clone/TabletScheduler.java

```diff
@@ -1185,8 +1185,7 @@ private void deleteReplicaInternal(TabletSchedCtx tabletCtx, Replica replica, St
             // process.
             sendDeleteReplicaTask(replica.getBackendId(), tabletCtx.getTabletId(), tabletCtx.getSchemaHash());
         }
-
```

Comment:
The backend id should be replica.getBackendId(), only one replica will be dropped from backend.

Reply:
updated